src/libraries/System.Formats.Tar/tests/TarFile/TarFile.CreateFromDirectory.File.Tests.cs

```diff
@@ -253,5 +254,50 @@ public void SkipRecursionIntoBaseDirectorySymlink()
 
             Assert.Null(reader.GetNextEntry());
         }
+
+        [ConditionalTheory(nameof(CanExtractWithTarTool))]
+        [MemberData(nameof(GetPaxAndGnuTestCaseNames))]
```

Comment:
As discussed via chat - Some link tests are failing, possibly because the OS does not support the creation of symlinks, so they should get separated into their own separate tests and decorated with the `CanCreateSymbolicLinks` condition.

Reply:
Addressed slightly different on latest commit:
https://github.com/dotnet/runtime/blob/1b4bf333f81e20c615c0a94e55c5605f750a7252/src/libraries/System.Formats.Tar/tests/TarFile/TarFile.CreateFromDirectory.File.Tests.cs#L267-L271

Reply:
Maybe, but won't this fail on any Windows version if `MountHelper.CanCreateSymbolicLinks` is false? I don't see it being checked in the current commit.

As you know symlinks on Windows require elevation [unless you have developer mode](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/). You almost certainly have developer mode enabled on your own machine, but it may not be enabled on others' machines. This won't show up in Helix because I believe they run all tests elevated, but any test we have that creates symlinks must either test for admin explicitly or test `MountHelper.CanCreateSymbolicLinks`

Reply:
> I don't see it being checked in the current commit.

Yeah, I removed it because nano was still failing (even with the `CanCreateSymbolicLinks` check). 
You can see the run for that commit:
https://github.com/dotnet/runtime/pull/75023/checks?check_run_id=8242739135

Got this error log:
https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-75023-merge-fbe5d4bb79764adabe/System.Formats.Tar.Tests/1/console.da17f2b2.log?helixlogtype=result

Reply:
I believe you, just saying that if you fix whatever that problem is, you'd still need that check

Reply:
How are we using CanCreateSymbolicLinks in our System.IO symlink tests?